api/src/Responder/Response/ApiResponse.php

```diff
@@ -187,4 +188,32 @@ protected function encode($data)
         $prettyPrint = $debug?JSON_PRETTY_PRINT:0;
         return json_encode($data, $prettyPrint);
     }
+
+    /**
+     * Creates a redirect response.
+     *
+     * @param  string  $url
+     * @param  int     $status
+     * @param  array   $headers
+     *
+     * @throws InvalidArgumentException
+     *
+     * @return $this
+     */
+    public function redirect($url, $status = 302, array $headers = [])
```

Comment:
This method should not accept headers, as there are other existing methods for adding headers. I prefer a stricter api.